extensions/shared/jetpack-likes-and-sharing-panel.js

```diff
@@ -12,7 +14,15 @@ JetpackLikesAndSharingPanel.Slot = () => (
 	<Slot>
 		{ fills => {
 			if ( ! fills.length ) {
-				return null;
+				const sharingAdminLink = '/wp-admin/admin.php?page=jetpack#/sharing';
```

Comment:
Might be good to add a note here that this doesn't need Calypso-URL version for WPCOM where the feature is always enabled?

Does this need to direct to Calypso for Atomic sites, tho?